portal-ui/src/screens/Console/License/License.tsx

```diff
@@ -49,23 +49,22 @@ const styles = (theme: Theme) =>
     },
     licenseContainer: {
       padding: "20px 52px 0px 28px",
-      background:
-        "transparent linear-gradient(180deg, #ffffff 0%, #d6e1e8 100%) 0% 0% no-repeat padding-box",
+      background: "#032F51",
       boxShadow: "0px 3px 7px #00000014",
       "& h2": {
-        color: "#000",
-        marginBottom: "50px",
+        color: "#FFF",
+        marginBottom: "67px",
```

Comment:
Please remove the px & leave it as int
```suggestion
        marginBottom: 67,
```